### PR TITLE
docs(changelog): includeEvents is what's coming in spinnaker 1.30

### PR DESCRIPTION
**Pull request by chawlabhishek**
### :pencil: Description
- Adds release notes for includeEvents parameter.

### CODE REVIEW

File: content/en/docs/releases/next-release-preview/index.md [hunk 1]
```diff
@@ -69,3 +69,10 @@ In this example, even though each trigger has its own artifact(s) defined, when
 - Replace deprecated Azure SDK `com.microsoft.azure:azure` with the new Azure SDK `com.azure.resourcemanager:azure-resourcemanager`
 - Add a new `AzureManagedImageCachingAgent` which is caching managed images from the specified region and resource group
 - Update `AzureVMImageLookupController` to return the newly cached managed images when `managedImages` query parameter is set to `true`
+
+### Addition of includeEvents parameter
+`includeEvents` has been added as a configurable parameter in orca that enables users to fetch [Kubernetes Events](https://kubernetes.io/docs/reference/kubernetes-api/cluster-resources/event-v1/) from clouddriver. This parameter defaults to `false`.
+
+![includeEvents](includeEvents.png)
```
Comment (dbyron-sf) [inline content/en/docs/releases/next-release-preview/index.md:76]: I'd still prefer we leave out the bit about setting up a pipeline parameter for this.  People build pipelines in lots of ways, and set fields in pipeline stages in lots of ways.  This doesn't feel like it really adds to the discussion of `includeEvents` specifically.

File: content/en/docs/releases/next-release-preview/index.md [hunk 2]
```diff
@@ -69,3 +69,10 @@ In this example, even though each trigger has its own artifact(s) defined, when
...
+
+This should be referenced in `pipeline.json` as `"includeEvents": "${parameters.includeEvents}"`. Once the parameter has been configured and the value is set to true then orca would retrieve the events back from clouddriver. This would however come with slight degradation in performance since the events would be retrieved with the manifest.
```
Comment (dbyron-sf) [inline content/en/docs/releases/next-release-preview/index.md:78]: What pipeline stages support `includeEvents`?

Comment (chawlabhishek) [reply content/en/docs/releases/next-release-preview/index.md]: @dbyron-sf Updated!

File: content/en/docs/releases/next-release-preview/index.md (hunk 1 above)
Comment (chawlabhishek) [reply content/en/docs/releases/next-release-preview/index.md]: @dbyron-sf Removed the configuration bit from the docs. Please lemme know if this is okay?

File: content/en/docs/releases/next-release-preview/index.md [hunk 3]
```diff
@@ -69,3 +69,8 @@ In this example, even though each trigger has its own artifact(s) defined, when
 - Replace deprecated Azure SDK `com.microsoft.azure:azure` with the new Azure SDK `com.azure.resourcemanager:azure-resourcemanager`
 - Add a new `AzureManagedImageCachingAgent` which is caching managed images from the specified region and resource group
 - Update `AzureVMImageLookupController` to return the newly cached managed images when `managedImages` query parameter is set to `true`
+
+### Addition of includeEvents parameter
+`includeEvents` has been added as a configurable parameter in orca that enables users to fetch [Kubernetes Events](https://kubernetes.io/docs/reference/kubernetes-api/cluster-resources/event-v1/) from clouddriver. This parameter defaults to `false` and is supported by `DeployManifestStage`, `ScaleManifestStage`, `DisableManifestStage`, `EnableManifestStage`, `PatchManifestStage`, `ResumeRolloutManifestStage` and `UndoRolloutManifestStage` stages.
```
Comment (dbyron-sf) [inline content/en/docs/releases/next-release-preview/index.md:74]:
```suggestion
`includeEvents` has been [added as a configurable parameter in orca](https://github.com/spinnaker/orca/pull/4301) that enables users to fetch [Kubernetes Events](https://kubernetes.io/docs/reference/kubernetes-api/cluster-resources/event-v1/) from clouddriver. This parameter defaults to `false` and is supported by `DeployManifestStage`, `ScaleManifestStage`, `DisableManifestStage`, `EnableManifestStage`, `PatchManifestStage`, `ResumeRolloutManifestStage` and `UndoRolloutManifestStage` stages.
```

File: content/en/docs/releases/next-release-preview/index.md [hunk 4]
```diff
@@ -69,3 +69,8 @@ In this example, even though each trigger has its own artifact(s) defined, when
...
+
+Once the parameter has been configured and the value is set to true then orca would retrieve the events back from clouddriver. This would however come with slight degradation in performance since the events would be retrieved with Manifest.
```
Comment (dbyron-sf) [inline content/en/docs/releases/next-release-preview/index.md:76]:
```suggestion
Once the parameter has been configured and the value is set to true then orca retrieves the events back from clouddriver. Be aware that the size of the pipeline execution context may (significantly) increase by including events.
```

**Review by dbyron-sf — APPROVED** [review]

File: content/en/docs/releases/next-release-preview/index.md (hunk 3 above)
Comment (chawlabhishek) [reply content/en/docs/releases/next-release-preview/index.md]: Updated!

File: content/en/docs/releases/next-release-preview/index.md (hunk 4 above)
Comment (chawlabhishek) [reply content/en/docs/releases/next-release-preview/index.md]: Updated!

**Pull request merged**